infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -241,7 +227,8 @@ public TokenType txnCanCommit(TxResolutionInfo txInfo, /** Input. */
             if (conflictParamSet != null && conflictParamSet.size() > 0) {
                 // for each key pair, check for conflict;
                 // if not present, check against the wildcard
-                conflictParamSet.forEach(conflictParam -> {
+                for (byte[] conflictParam : conflictParamSet) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS135)